mkosi/__init__.py

```diff
@@ -5697,7 +5698,7 @@ def load_args(args: argparse.Namespace) -> CommandLineArguments:
         args.cache_pre_dev = None
         args.cache_pre_inst = None
 
-    args.output = os.path.abspath(args.output)
+    args.output = os.path.abspath(str(args.output))
```

Comment:
This doesn't look correct. `args.output` is a string already, no? And if it *wasn't* a string, this conversion would only hide the issue.

Reply:
This seemingly fixed itself in one of the newer pyright releases it seems. Removed the `str` again